google/cloud/spanner/value_test.cc

```diff
@@ -904,6 +912,75 @@ TEST(Value, CommitTimestamp) {
   EXPECT_FALSE(bad.ok());
 }
 
+class ValueStreamOperator
+    : public TestWithParam<std::tuple<std::string, Value, const char*>> {
+ public:
+  static std::string TestName(
+      TestParamInfo<ValueStreamOperator::ParamType> const& info) {
+    return std::get<0>(info.param);
+  }
+};
+
+TEST_P(ValueStreamOperator, ValueType) {
+  auto param = GetParam();
+  Value value_int64(42);
+  std::stringstream scalar_value_stream;
+  scalar_value_stream << std::get<1>(param);
+  EXPECT_STREQ(scalar_value_stream.str().c_str(), std::get<2>(param));
+}
+
+INSTANTIATE_TEST_SUITE_P(
+    ValueStreamOperatorScalar, ValueStreamOperator,
+    testing::Values(
+        std::make_tuple("Bool", Value(false), "false"),
+        std::make_tuple("Int64", Value(42), "42"),
+        std::make_tuple("NullInt64", MakeNullValue<std::int64_t>(), ""),
+        std::make_tuple("Float64", Value(42.0), "42.000000"),
+        std::make_tuple("InfinityFloat64",
+                        Value(std::numeric_limits<double>::infinity()),
+                        "Infinity"),
+        std::make_tuple("NegativeInfinityFloat64",
+                        Value(-std::numeric_limits<double>::infinity()),
+                        "-Infinity"),
+        std::make_tuple("NaNFloat64", Value(std::nan("NaN")), "NaN"),
+        std::make_tuple("String", Value("Seatac Astronomy"),
+                        "Seatac Astronomy"),
+        std::make_tuple("Bytes", Value("DEADBEEF"), "DEADBEEF"),
+        std::make_tuple(
+            "Timestamp",
+            Value(MakeTimestamp(MakeTimePoint(1561147549LL, 0)).value()),
+            "2019-06-21T20:05:49Z"),
+        std::make_tuple("Date", Value(Date(1970, 1, 1)), "1970-01-01")),
+
```

Comment:
Nit: Delete blank line to match other instantiations (or vice versa)?

Reply:
Done.